packages/enhanced-img/types/index.d.ts

```diff
@@ -0,0 +1,3 @@
+import type { Plugin } from 'vite';
+
+export function staticImages(): Promise<Plugin[]>;
```

Comment:
would it be overkill to use `dts-buddy` here to prevent this sort of thing?

```suggestion
export function enhancedImages(): Promise<Plugin[]>;
```